pcm.cpp

```diff
@@ -836,9 +836,9 @@ void print_basic_metrics_csv(const PCM * m, const State & state1, const State &
     if (m->isActiveRelativeFrequencyAvailable())
         cout << ',' << getActiveRelativeFrequency(state1, state2);
     if (m->isL3CacheMissesAvailable())
-        cout << ',' << float_format(getL3CacheMisses(state1, state2));
```

Comment:
this normalizes by millions: https://software.intel.com/content/www/us/en/develop/blogs/intel-pcm-column-names-decoder-ring.html